scripts/ci/run-ci.sh

```diff
@@ -30,24 +30,9 @@ cmake --build --preset=${CMAKE_PRESET}
 # Require regression-like tests to be enabled and pass
 export CELER_TEST_STRICT=1
 
-# Set up test arguments
-CTEST_TOOL=Test
-CTEST_ARGS=""
-case ${CMAKE_PRESET} in
-  vecgeom-demos )
-    CTEST_ARGS="-L app"
-    ;;
-  valgrind )
-    CTEST_TOOL="MemCheck"
-    CTEST_ARGS="-LE nomemcheck"
-    ;;
-  * )
-    ;;
-esac
-
```

Comment:
We haven't had a valgrind job on jenkins for a while.

Reply:
Wouldn't this be an occasion to reintroduce it or it is known not to work?

Reply:
No this is not the occasion; that could be a separate PR if it's needed. I don't see any advantage since we have the clang address sanitizer.